cmd/launcher/launcher.go

```diff
@@ -179,33 +179,31 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 		"build", versionInfo.Revision,
 	)
 
-	var controlService *control.ControlService
-	var runner *desktopRunner.DesktopUsersProcessesRunner
-	if opts.KolideServerURL == "k2device-preprod.kolide.com" || opts.KolideServerURL == "localhost:3443" {
-		// If the control server has been opted-in to, run the control service
-		controlService, err = createControlService(ctx, logger, opts)
-		if err != nil {
-			return fmt.Errorf("failed to setup control service: %w", err)
-		}
-		runGroup.Add(controlService.Execute, controlService.Interrupt)
-
-		// serverDataBucketConsumer handles server data table updates
-		serverDataBucketConsumer := control.NewBucketConsumer(logger, db, osquery.ServerProvidedDataBucket)
-		controlService.RegisterConsumer("kolide_server_data", serverDataBucketConsumer)
-
-		runner = desktopRunner.New(
-			desktopRunner.WithLogger(logger),
-			desktopRunner.WithUpdateInterval(time.Second*5),
-			desktopRunner.WithHostname(opts.KolideServerURL),
-			desktopRunner.WithAuthToken(ulid.New()),
-			desktopRunner.WithUsersFilesRoot(rootDirectory),
-		)
-		runGroup.Add(runner.Execute, runner.Interrupt)
-
-		if controlService != nil {
-			controlService.RegisterConsumer("desktop", runner)
-		}
+	controlService, err := createControlService(ctx, logger, opts)
+	if err != nil {
+		return fmt.Errorf("failed to setup control service: %w", err)
 	}
+	runGroup.Add(controlService.Execute, controlService.Interrupt)
+
+	// serverDataBucketConsumer handles server data table updates
+	serverDataBucketConsumer := control.NewBucketConsumer(logger, db, osquery.ServerProvidedDataBucket)
+	controlService.RegisterConsumer("kolide_server_data", serverDataBucketConsumer)
+
+	desktopFlagsBucketConsumer := control.NewBucketConsumer(logger, db, "kolide_desktop_flags")
+	controlService.RegisterConsumer("kolide_desktop_flags", desktopFlagsBucketConsumer)
```

Comment:
```suggestion
	desktopFlagsBucketConsumer := control.NewBucketConsumer(logger, db, "agent_flags")
	controlService.RegisterConsumer("agent_flags", desktopFlagsBucketConsumer)
```

I think we should make this for all the agent flags.